docs/tutorial/security.md

```diff
@@ -27,18 +27,8 @@ see [SECURITY.md](https://github.com/electron/electron/tree/master/SECURITY.md)
 
 ## Chromium Security Issues and Upgrades
 
-While Electron strives to support new versions of Chromium as soon as possible,
-developers should be aware that upgrading is a serious undertaking - involving
-hand-editing dozens or even hundreds of files. Given the resources and
-contributions available today, Electron will often not be on the very latest
-version of Chromium, lagging behind by several weeks or a few months.
-
-We feel that our current system of updating the Chromium component strikes an
-appropriate balance between the resources we have available and the needs of
-the majority of applications built on top of the framework. We definitely are
-interested in hearing more about specific use cases from the people that build
-things on top of Electron. Pull requests and contributions supporting this
-effort are always very welcome.
+Electron keeps up to date with alternating Chromium releases. For more information,
+see the [Electron Release Cadence blog post](https://electronjs.org/blog/12-week-cadence).
```

Comment:
Isn't this change already being handled in another PR? :thinking: It seems off-topic here and ought to be removed

Reply:
Maybe wrong base branch🤔

Reply:
Yes 😅 
Idk how I messed that up. Sorry guys. Is there a way to change / fix this? 
These changes were already added to master so I'm not sure why they show up (https://github.com/electron/electron/pull/19804)

Reply:
merging or rebasing w / master ought to work

Reply:
...actually, you seem to have resolved this in yesterday's commits.